graph/graph.go

```diff
@@ -33,71 +36,53 @@ func borderHorizontal(out *string, width int) {
 	}
 }
 
-// GenerateGraph with ASCII graph with ANSI escapes
-func GenerateGraph(chart *api.Chart, width int, height int, chartTheme ChartTheme, timezone *time.Location) (string, error) {
-	out := "┏"
-	maxSize := len(strings.Split(chart.High.String(), ".")[0]) + 3
-	borderHorizontal(&out, width+maxSize+3)
-	out += "┓"
-	colour := 92
-	if chart.Length < width/5 {
-		chart.Length = width / 3
-	}
-	if chart.Length > width {
-		mod := 2 * (chart.Length / width)
-		chartTemp := make([]*api.Bar, 0)
-		for i, bar := range chart.Bars {
-			if (i+1)%mod == 0 {
-				chartTemp = append(chartTemp, bar)
-			}
+func GenerateBraille(chart *api.Chart, width int, height int) string {
```

Comment:
I'm unsure why there's a need to delegate generation of the braile theme to a new function. The rest of the themes just replace the relevant characters in a condtional as seen [here](https://github.com/ericm/stonks/blob/1e9523c54994076b7711e0325329fd02b7594290/graph/graph.go#L107). Could it be refactored to do the same?

Reply:
The braille graph is drawn in an entirely different way from the other graph themes so I don't think the same result can be achieved with just a character swap.

Reply:
Could you add it to the flag `-t`. I'm also not sure about delegating to an unmaintaned fork to draw braille considering the `graph` package is supposed to contain the generation code.

Reply:
Is the use of drawille-go a complete deal breaker? If so I can attempt to re implement the drawing without a library.